www/docs/src/styles/globals.css

```diff
@@ -195,6 +195,7 @@ html {
 
 .sl-markdown-content h3 {
   font-size: 1rem;
+  padding-top: 1rem;
```

Comment:
Here I'm giving some breathing space for `h3` tags as I find that everything is super packed and makes it for a difficult to read docs in general when you're using `h3` more often.